app/views/shared/_aside_toggle.html.haml

```diff
@@ -1,7 +1,7 @@
 - if @search.count > 0
   .toggle.mobile.Marticle
-    %span.icon-arrow-thin-left{"aria-hidden" => "true"}
-    Refine
+    Refine search
+    %span.icon-arrow-thin-right{"aria-hidden" => "true"}
```

Comment:
Changing the arrow from left to right corrects an inconsistency in the user interface.